src/app/zap-templates/zcl/data-model/chip/refrigerator-alarm.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+  <bitmap name="AlarmMap" type="BITMAP32">
+     <cluster code="0x0057" />
+     <item name="DoorOpen" mask="0x01"/>
```

Comment:
Python codegen for this seems to have done:

```python

    class Bitmaps:
        class AlarmMap(IntFlag):
```

with empty content.

I believe the actual name for this should be `field` not `item` based on other examples

Reply:
@bzbarsky-apple @tecimovic is there a way to enforce xml validity? I believe we have a linter however we never enabled it. Would the linter have caught this?

Reply:
I don't think we have a sane schema right now for this format...

Reply:
> I believe the actual name for this should be field not item based on other examples

Updated